modules/010-prometheus-crd/crds/customalertmanager.yaml

```diff
@@ -1249,3 +1267,13 @@ spec:
           name: Type
           description: 'Type of an Alertmanager.'
           type: string
+        - jsonPath: .metadata.generation
```

Comment:
`.metadata.generation` is not the last observed. It is just a generation. The last observed should be the one from the status. It is not guaranteed that Deckhouse actually observed the generation because it may not be running at the time.